### PR TITLE
refactor pkg structure and impl ghost service

### PR DESCRIPTION
**Pull request by junbeomlee**
## Related Issue

resolve: #*(Write down your issue number)*

refactor pkg structure and impl ghost service

## Checklist

Thank you for your contribution.
Before submitting this PR, please make sure:

- [x] Test case
- [x] End of Work

### CODE REVIEW

File: server/src/app/ghost/ghost.service.ts [hunk 1]
```diff
@@ -1,4 +1,28 @@
...
+      {
+        userId,
+      },
+    );
+  }
+
+  async findAll(page: number): Promise<Ghost[]> {
+    return await this.ghostRepository.find({
+      take: 25,
+      skip: 25 * (page - 1),
```
Comment (hihiboss) [inline server/src/app/ghost/ghost.service.ts:28]: You implemented `skip: 25 * (page - 1)`, but in your test code, you tested with `findAll(0)`. In tc, `skip = -25`. Please check this.

File: server/src/app/transaction/transaction.service.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import {Transaction} from '../../domain/transaction/transaction.entity';
-import {CreateTransactionRequest} from './request/create-transaction.request';
+import {CreateTransactionDto} from './dto/create-transaction.dto';
 
 export interface ITransactionService {
-    create(newTransactionRequest: CreateTransactionRequest): Promise<Transaction>;
+    create(newTransactionRequest: CreateTransactionDto): Promise<Transaction>;
```
Comment (hihiboss) [inline server/src/app/transaction/transaction.service.ts]: Please change variable name :)

**Comment by junbeomlee** [comment]
fixed it!

**Review by hihiboss — APPROVED** [review]
LGTM :)

**Pull request merged**